### PR TITLE
Update IO rename form, refs# 13530

### PR DESCRIPTION
**Pull request by sbreker**
- Removed modal and replaced with alert class
- Updated to use b5 checkbox classes
- Updated layout using b5 columns

### CODE REVIEW

File: plugins/arDominionB5Plugin/modules/informationobject/templates/renameSuccess.php [hunk 1]
```diff
@@ -47,36 +30,64 @@
...
-                ->help(__('Do not use any special characters or spaces in the slug - only lower case alphanumeric characters (a-z, 0-9) and dashes (-) will be saved. Other characters will be stripped out or replaced. Editing the slug will not automatically update the other fields.')), $resource); ?>
-
-            <p><?php echo __('Original slug'); ?>: <em><?php echo $resource->slug; ?></em></p>
+            <hr />
+
+            <div class="row mb-3">
+              <div class="col-md-3">
+                <div class="rename-form-field-toggle form-check">
+                  <input class="form-check-input" type="checkbox" id="rename_enable_title" checked>
+                  <label class="form-check-label" for="flexCheckChecked">
```
Comment (jraddaoui) [inline plugins/arDominionB5Plugin/modules/informationobject/templates/renameSuccess.php]: `for="rename_enable_title"`

File: plugins/arDominionB5Plugin/modules/informationobject/templates/renameSuccess.php [hunk 2]
```diff
@@ -47,36 +30,64 @@
...
+            <hr />
+
+            <div id="rename-slug-warning" class="alert alert-danger" style="display: none;">
+              <?php echo __('A slug based on this title already exists so a number has been added to pad the slug.'); ?>
+            </div>
+            <div class="row mb-3">
+              <div class="col-md-3">
+                <div class="rename-form-field-toggle form-check">
+                  <input class="form-check-input" type="checkbox" id="rename_enable_slug" checked>
+                  <label class="form-check-label" for="flexCheckChecked">
```
Comment (jraddaoui) [inline plugins/arDominionB5Plugin/modules/informationobject/templates/renameSuccess.php]: `for="rename_enable_slug"`

File: plugins/arDominionB5Plugin/modules/informationobject/templates/renameSuccess.php [hunk 3]
```diff
@@ -47,36 +30,64 @@
...
+              <div class="col-md-8 offset-md-1">
+                <?php echo render_field($form->title
+                    ->label(__('Title'))
+                    ->help(__('Editing the description title will automatically update the slug field if the "Update slug" checkbox is selected - you can still edit it after.')), $resource); ?>
+              </div>
+              <p><?php echo __('Original title'); ?>: <em><?php echo $resource->title; ?></em></p>
+            </div>
+            <hr />
+
+            <div id="rename-slug-warning" class="alert alert-danger" style="display: none;">
```
Comment (jraddaoui) [inline plugins/arDominionB5Plugin/modules/informationobject/templates/renameSuccess.php]: Add `role="alert"`.

File: plugins/arDominionB5Plugin/modules/informationobject/templates/renameSuccess.php [hunk 4]
```diff
@@ -47,36 +30,64 @@
...
-                  ->help(__('Do not use any special characters or spaces in the filename - only lower case alphanumeric characters (a-z, 0-9) and dashes (-) will be saved. Other characters will be stripped out or replaced. Editing the filename will not automatically update the other fields.')), $resource); ?>
-
-              <p><?php echo __('Original filename'); ?>: <em><?php echo $resource->digitalObjectsRelatedByobjectId[0]->name; ?></em></p>
-
+              <hr />
+              <div class="row mb-3">
+                <div class="col-md-3">
+                  <div class="rename-form-field-toggle form-check">
+                    <input class="form-check-input" type="checkbox" id="rename_enable_filename" checked>
+                    <label class="form-check-label" for="flexCheckChecked">
```
Comment (jraddaoui) [inline plugins/arDominionB5Plugin/modules/informationobject/templates/renameSuccess.php]: `for="rename_enable_filename"`

File: plugins/arDominionB5Plugin/scss/_layout.scss [hunk 5]
```diff
@@ -34,7 +34,7 @@ body {
 
 .dropdown-menu {
   box-shadow: $box-shadow;
-  
+
```
Comment (jraddaoui) [inline plugins/arDominionB5Plugin/scss/_layout.scss]: Ups ... thanks!

**Review by jraddaoui — CHANGES_REQUESTED** [review]
Looking great @sbreker!

With the dividers you added, I'd prefer to have each input on a single row, giving more space to fields like these ones that could be long, and avoiding all the whitespace under the check-boxes.

Added a few minor but important comments below.

File: plugins/arDominionB5Plugin/scss/_layout.scss (hunk 5 above)
Comment (jraddaoui) [reply plugins/arDominionB5Plugin/scss/_layout.scss]: Added this in a different commit, please rebase ;)

Comment (sbreker) [reply plugins/arDominionB5Plugin/scss/_layout.scss]: 👍

File: plugins/arDominionB5Plugin/modules/informationobject/templates/renameSuccess.php (hunk 1 above)
Comment (sbreker) [reply plugins/arDominionB5Plugin/modules/informationobject/templates/renameSuccess.php:39]: 🤦

File: plugins/arDominionB5Plugin/modules/informationobject/templates/renameSuccess.php (hunk 4 above)
Comment (sbreker) [reply plugins/arDominionB5Plugin/modules/informationobject/templates/renameSuccess.php]: Fixed

File: plugins/arDominionB5Plugin/modules/informationobject/templates/renameSuccess.php (hunk 3 above)
Comment (sbreker) [reply plugins/arDominionB5Plugin/modules/informationobject/templates/renameSuccess.php]: Done

File: plugins/arDominionB5Plugin/modules/informationobject/templates/renameSuccess.php (hunk 2 above)
Comment (sbreker) [reply plugins/arDominionB5Plugin/modules/informationobject/templates/renameSuccess.php]: Fixed

File: plugins/arDominionB5Plugin/modules/informationobject/templates/renameSuccess.php (hunk 1 above)
Comment (sbreker) [reply plugins/arDominionB5Plugin/modules/informationobject/templates/renameSuccess.php]: Fixed

**Comment by sbreker** [comment]
Hi @jraddaoui - thanks for the feedback! The layout has been updated to remove the columns bringing the fields all over to the left side to give the long text input fields more room.

**Review by jraddaoui — APPROVED** [review]
Nice, thanks @sbreker!!

**Pull request merged**